slides/k8s/accessinternal.md

```diff
@@ -0,0 +1,130 @@
+# Accessing internal services
+
+- When we are logged in on a cluster node, we can access internal services
+
+  (by virtue of the CNI model: all nodes can reach all pods and services)
+
+- When we are accessing a remote cluster, things are different
+
+  (generally, our local machine won't have access to the cluster's internal subnet)
+
+- How can we temporarily access a service without exposing it to everyone?
+
+--
+
+- `kubectl proxy`: gives us access to the API, which includes a proxy for HTTP resources
+
+- `kubectl port-forward`: allows forwarding of TCP ports to arbitrary pods, services, ...
+
+---
+
+## Suspension of disbelief
+
+The exercises in this section assume that we have set up `kubectl` on our
+local machine in order to access a remote cluster.
+
+We will therefore show how to access services and pods of the remote cluster,
+from our local machine.
+
+You can also run these exercises directly on the cluster (if you haven't
+installed and set up `kubectl` locally).
+
+It will be less useful (since you could access services and pods directly),
```

Comment:
Maybe say "Running commands locally" instead of "It", for clarity?

Reply:
Absolutely!